staging/src/k8s.io/apiserver/pkg/server/routes/openapi.go

```diff
@@ -32,8 +32,15 @@ type OpenAPI struct {
 
 // Install adds the SwaggerUI webservice to the given mux.
 func (oa OpenAPI) Install(c *restful.Container, mux *mux.PathRecorderMux) {
+	// NOTE: [DEPRECATION] We will announce deprecation for format-separated endpoints for OpenAPI spec,
+	// and switch to a single /openapi/v2 endpoint in Kubernetes 1.10. The design doc and deprecation process
```

Comment:
1.10 or 1.11?

Reply:
The deprecation plan in design doc is to announce deprecation in 1.10, redirect old endpoint to new endpoint in 1.11, and remove old endpoint (and probably return a deprecation message) in 1.12